pom.xml

```diff
@@ -91,7 +91,7 @@
                             <value>conf/log4j.properties</value>
                         </property>
                     </systemProperties>
-                    <argLine>-javaagent:${basedir}/target/dd-java-agent-${dd-java-agent-version}.jar ${argLine} -Xms512m -Xmx3500m -DsurefireForkNumber=${surefire.forkNumber}</argLine>
+                    <argLine>-javaagent:${basedir}/target/dd-java-agent-0.69.0-SNAPSHOT.jar ${argLine} -Xms512m -Xmx3500m -DsurefireForkNumber=${surefire.forkNumber}</argLine>
```

Comment:
the `target` folder is ephemeral, and it will be removed if the `mvn clean` phase is executed.
I'd consider using another folder to keep the java agent.

Reply:
`git mv target/dd-java-agent-0.69.0-SNAPSHOT.jar .dd`